ANSWERS.md

```diff
@@ -1,14 +1,18 @@
 **1. List all of the main states a process may be in at any point in time on a standard Unix system. Briefly explain what each of these states means.**
 
-
+- Running - the process is active, or ready
```

Comment:
These are good, you could have also mentioned created, the initial process state and "ready" when the process is loaded into memory and preparing to be executed